packages/config/config/devices/0x031e/vzw32-sn.json

```
@@ -0,0 +1,1013 @@
+			"$import": "templates/inovelli_templates.json#led_effect_level",
+			"label": "All LED Strip Effect - Level"
+		},
+		{
+			"#": "99[0xff0000]",
+			"$import": "templates/inovelli_templates.json#led_effect_color",
+			"label": "All LED Strip Effect - Color"
+		},
+		{
+			"#": "99[0xff000000]",
+			"$import": "templates/inovelli_templates.json#all_led_effect_effect",
+			"label": "All LED Strip Effect - Effect"
+		},
+		{
+			"#": "100",
+			"label": "LED Brightness Scaling",
+			"valueSize": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Match VZM31 (Blue series)",
+					"value": 0
+				},
+				{
+					"label": "Match LZW31 (Red series)",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "101",
+			"label": "mmWave Height Minimum (Floor)",
+			"valueSize": 2,
+			"unit": "cm",
+			"minValue": -600,
+			"maxValue": 599,
+			"defaultValue": -600
+		},
+		{
+			"#": "102",
+			"label": "mmWave Height Maximum (Ceiling)",
+			"valueSize": 2,
+			"unit": "cm",
+			"minValue": -599,
+			"maxValue": 600,
+			"defaultValue": 600
+		},
+		{
+			"#": "103",
+			"label": "mmWave Width Minimum (Left)",
+			"valueSize": 2,
+			"unit": "cm",
+			"minValue": -600,
+			"maxValue": 599,
+			"defaultValue": -600
+		},
+		{
+			"#": "104",
+			"label": "mmWave Width Maximum (Right)",
+			"valueSize": 2,
+			"unit": "cm",
+			"minValue": -599,
+			"maxValue": 600,
+			"defaultValue": 600
+		},
+		{
+			"#": "105",
+			"label": "mmWave Depth Minimum (Near)",
+			"valueSize": 2,
+			"unit": "cm",
+			"minValue": 0,
+			"maxValue": 599,
+			"defaultValue": 0
+		},
+		{
+			"#": "106",
+			"label": "mmWave Depth Maximum (Far)",
+			"valueSize": 2,
+			"unit": "cm",
+			"minValue": 1,
+			"maxValue": 600,
+			"defaultValue": 600
+		},
+		{
+			"#": "108",
+			"label": "mmWave Stay Life",
+			"description": "1 = 50 ms, 20 = 1 second, 300 = 15 seconds",
+			"valueSize": 4,
+			"unit": "50 ms",
+			"minValue": 0,
+			"maxValue": 4294967295,
+			"defaultValue": 300,
+			"unsigned": true
+		},
+		{
+			"#": "109[0xff000000]",
+			"label": "After Hours Time Range Start Hour",
+			"description": "The UTC hour where After Hours mode should start",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 59,
+			"defaultValue": 0
+		},
+		{
+			"#": "109[0xff0000]",
+			"label": "After Hours Time Range Start Minute",
+			"description": "The minute where After Hours mode should start",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 23,
+			"defaultValue": 0
+		},
+		{
+			"#": "109[0xff00]",
+			"label": "After Hours Time Range End Hour",
+			"description": "The UTC hour where After Hours mode should end",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 59,
+			"defaultValue": 0
+		},
+		{
+			"#": "109[0xff]",
+			"label": "After Hours Time Range End Minute",
+			"description": "The minute where After Hours mode should end",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 23,
+			"defaultValue": 0
+		},
```

Looks like a bit of a mixup:
```suggestion
		{
			"#": "109[0xff000000]",
			"label": "After Hours Time Range Start Hour",
			"description": "The UTC hour where After Hours mode should start",
			"valueSize": 4,
			"minValue": 0,
			"maxValue": 23,
			"defaultValue": 0
		},
		{
			"#": "109[0xff0000]",
			"label": "After Hours Time Range Start Minute",
			"description": "The minute where After Hours mode should start",
			"valueSize": 4,
			"minValue": 0,
			"maxValue": 59,
			"defaultValue": 0
		},
		{
			"#": "109[0xff00]",
			"label": "After Hours Time Range End Hour",
			"description": "The UTC hour where After Hours mode should end",
			"valueSize": 4,
			"minValue": 0,
			"maxValue": 23,
			"defaultValue": 0
		},
		{
			"#": "109[0xff]",
			"label": "After Hours Time Range End Minute",
			"description": "The minute where After Hours mode should end",
			"valueSize": 4,
			"minValue": 0,
			"maxValue": 59,
			"defaultValue": 0
		},
```